docs/Database_Schema.md

```diff
@@ -224,3 +224,74 @@ CREATE TABLE IF NOT EXISTS information (
     PRIMARY KEY(key)
 )
 ```
+
+----
+
+## 9. Table **transactionPool**
+
+### _Schema_
+
+| Column            | Data Type | PK | Not NULL | Default  |Description|
+|:----------------- |:--------- |:--:|:--------:| -------- | --------- |
+| tx_hash           | BLOB      | Y  | Y        |          | The hash of transaction |
+| type              | INTEGER   |    | Y        |          | The type of transaction |
+| payload           | BLOB      |    | Y        |          | The transaction data payload |
+
+### _Create Script_
+
+```sql
+CREATE TABLE IF NOT EXISTS "transactionPool" (
+    "tx_hash"               BLOB    NOT NULL,
+    "type"                  INTEGER NOT NULL,
+    "payload"               BLOB    NOT NULL,
+    PRIMARY KEY("tx_hash")
+)
+```
+
+----
+
+## 4. Table **txInputPool**
+
+### _Schema_
+
+| Column            | Data Type | PK | Not NULL | Default  |Description|
+|:----------------- |:--------- |:--:|:--------:| -------- | --------- |
+| tx_hash           | BLOB      | Y  | Y        |          | The hash of transaction|
+| utxo              | BLOB      | Y  | Y        |          | The hash of the UTXO to be spent|
+| signature         | BLOB      |    | Y        |          | The signature of this transaction input|
+### _Create Script_
+
+```sql
+CREATE TABLE IF NOT EXISTS "txInputPool" (
+    "tx_hash"               BLOB    NOT NULL,
+    "utxo"                  BLOB    NOT NULL,
+    "signature"             BLOB    NOT NULL,
+    PRIMARY KEY("tx_hash","utxo")
+)
+```
+
+----
+
+## 5. Table **txOutputPool**
+
+### _Schema_
+
+| Column            | Data Type | PK | Not NULL | Default  |Description|
+|:----------------- |:--------- |:--:|:--------:| -------- | --------- |
+|  tx_hash          | BLOB      | Y  | Y        |          | The hash of transaction |
+|  output_index     | INTEGER   | Y  | Y        |          | The index of output in the outputs.|
```

Comment:
Why do we need output_index?

Reply:
The outputs is an array.
A key is required to distinguish.

Reply:
OKay